biggraphite/drivers/cassandra.py

```diff
@@ -1160,6 +1160,7 @@ def __init__(
 
         self.metadata_enabled = enable_metadata
 
+    @_utils.trace_accessor_func
```

Comment:
The name of the decorator does not seem explicit to me.

* `_utils` is very generic
* `_utils` stands for a private package / usage, it does not seem to be the intent here (can be used widely in the project)
* the function name could be imperative

My suggestion (feel free to suggest something else):
```
@tracing.trace
```